### PR TITLE
Details URI

### PR DESCRIPTION
**Pull request by aitorsanz**

### CODE REVIEW

File: team/src/main/resources/static/js/app-submit-single.js [hunk 1]
```diff
@@ -19,11 +19,11 @@ $(document).ready(
                             + "'>"
                             + "Get QR here!"
                             + "</a></div>"+
-                            "<div class='alert alert-success lead'><a target='_blank' href='"
+                            "<h3>Show Details<h3><div class='alert alert-info lead'><a target='_blank' href='"
                             + msg.uri+"/+'"
                             + ">"
                             + msg.uri+"/+"
-			    + "</a></div>");
```
Comment (juan-vg) [inline team/src/main/resources/static/js/app-submit-single.js:26]: You should not delete this

File: team/src/main/resources/static/js/app-submit-single.js [hunk 2]
```diff
@@ -19,11 +19,11 @@ $(document).ready(
                             + "'>"
                             + "Get QR here!"
                             + "</a></div>"+
-                            "<div class='alert alert-success lead'><a target='_blank' href='"
+                            "<h3>Show Details<h3><div class='alert alert-info lead'><a target='_blank' href='"
```
Comment (juan-vg) [inline team/src/main/resources/static/js/app-submit-single.js]: You should close the h3 tag

File: demo/src/main/java/urlshortener/demo/config/PersistenceContext.java [hunk 3]
```diff
@@ -10,6 +10,7 @@
 import urlshortener.common.repository.ShortURLRepository;
 import urlshortener.common.repository.ShortURLRepositoryImpl;
 
+@SuppressWarnings("ALL")
```
Comment (juan-vg) [inline demo/src/main/java/urlshortener/demo/config/PersistenceContext.java]: Why is needed?

File: team/src/main/java/urlshortener/zaratech/config/PersistenceContext.java [hunk 4]
```diff
@@ -10,6 +10,7 @@
 import urlshortener.common.repository.ShortURLRepository;
 import urlshortener.common.repository.ShortURLRepositoryImpl;
 
+@SuppressWarnings("ALL")
```
Comment (juan-vg) [inline team/src/main/java/urlshortener/zaratech/config/PersistenceContext.java]: Why is needed?

File: team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java [hunk 5]
```diff
@@ -14,6 +14,8 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 
+import java.sql.*;
```
Comment (juan-vg) [inline team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java]: Why is needed?

File: team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java [hunk 6]
```diff
@@ -29,17 +31,18 @@
...
 		return super.redirectTo(id, request);
-	}	  
-    
+	}
 
-	@Override
+
+
+    @Override
 	public ResponseEntity<ShortURL> singleShortener(@RequestParam("url") String url,
```
Comment (juan-vg) [inline team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java]: Tab here -> 4 spaces

File: team/src/main/java/urlshortener/zaratech/web/UrlShortnerWebController.java [hunk 7]
```diff
@@ -1,17 +1,29 @@
...
+import urlshortener.common.domain.ShortURL;
+import urlshortener.common.repository.ClickRepository;
+import urlshortener.common.repository.ShortURLRepository;
+import urlshortener.common.repository.ShortURLRepositoryImpl;
+
 
 @Controller
 public class UrlShortnerWebController {
-    
+    @Autowired
```
Comment (juan-vg) [inline team/src/main/java/urlshortener/zaratech/web/UrlShortnerWebController.java:21]: Is really needed?

File: team/src/main/java/urlshortener/zaratech/web/UrlShortnerWebController.java [hunk 8]
```diff
@@ -1,17 +1,29 @@
...
+    protected ShortURLRepository shortURLRepository;
+    @Autowired
+    protected ClickRepository clickRepository;
     @RequestMapping(value = {"", "/", "/single"}, method = RequestMethod.GET)
```
Comment (juan-vg) [inline team/src/main/java/urlshortener/zaratech/web/UrlShortnerWebController.java:26]: Blank line needed here

File: team/src/main/resources/static/js/app-submit-single.js (hunk 2 above)
Comment (juan-vg) [inline team/src/main/resources/static/js/app-submit-single.js]: You should close the h3 tag

File: team/src/main/webapp/WEB-INF/jsp/details.jsp [hunk 9]
```diff
@@ -11,6 +11,32 @@
 </head>
 <body>
 	<div class="container-full">
+		<nav class="navbar navbar-default">
```
Comment (juan-vg) [inline team/src/main/webapp/WEB-INF/jsp/details.jsp:14]: Tabs here -> 4 spaces

**Review by juan-vg — COMMENTED** [review]
Check this please

File: demo/src/main/java/urlshortener/demo/config/PersistenceContext.java (hunk 3 above)
Comment (aitorsanz) [reply demo/src/main/java/urlshortener/demo/config/PersistenceContext.java]: Suppress alert of protected JdbcTemplate jdbc;

File: team/src/main/java/urlshortener/zaratech/config/PersistenceContext.java (hunk 4 above)
Comment (aitorsanz) [reply team/src/main/java/urlshortener/zaratech/config/PersistenceContext.java]: Suppress alert of protected JdbcTemplate jdbc;

File: team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java (hunk 5 above)
Comment (aitorsanz) [reply team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java]: import.java.sql.* is not neccesary
Others have dependencies in code

File: demo/src/main/java/urlshortener/demo/config/PersistenceContext.java (hunk 3 above)
Comment (juan-vg) [reply demo/src/main/java/urlshortener/demo/config/PersistenceContext.java]: Suppress only that concrete warning then, please

File: team/src/main/java/urlshortener/zaratech/config/PersistenceContext.java (hunk 4 above)
Comment (juan-vg) [reply team/src/main/java/urlshortener/zaratech/config/PersistenceContext.java]: Suppress only that concrete warning then, please

File: team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java (hunk 5 above)
Comment (juan-vg) [reply team/src/main/java/urlshortener/zaratech/web/UrlShortenerControllerWithLogs.java]: no longer needed, I suppose

**Pull request merged**